kia-lib/src/test/kotlin/com/faforever/ice/IceAdapterIT.kt

```diff
@@ -190,5 +197,139 @@ class IceAdapterIT {
         val result = client2.receiveLobbyData()
 
         assertArrayEquals(data, result)
+        client1.stop()
```

Comment:
When I added a second test it was failing because the ports for the fake game clients in this first test were already in use.